python/README.md

```diff
@@ -36,11 +36,13 @@ AZURE_OPENAI_API_KEY=""
 import asyncio
 import semantic_kernel as sk
 from semantic_kernel.connectors.ai.open_ai import OpenAIChatCompletion, AzureChatCompletion
+from semantic_kernel.prompt_template.prompt_template_config import PromptTemplateConfig
+from semantic_kernel.utils.settings import openai_settings_from_dot_env, azure_openai_settings_from_dot_env
 
 kernel = sk.Kernel()
```

Comment:
I prefer that this is now also changes to `kernel = Kernel()` with `from semantic_kernel import Kernel`